src/Bex/Behat/ScreenshotExtension/Event/ScreenshotUploaderEvent.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Bex\Behat\ScreenshotExtension\Event;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * Dispatches an event when screenshots are uploaded.
+ */
+class ScreenshotUploaderEvent extends Event
```

Comment:
The name suggests that it's a generic uploader event. I would rather choose a more specific name that represents the actual action better. Something like:
- `ScreenshotUploadedEvent`
- `ScreenshotUploadCompleteEvent`

Reply:
> Nice job! 👍

Thank you. I am updating https://github.com/cawolf/behat-cucumber-formatter/pull/12

I am not a Symphony developer, so may need some help and ideas to avoid the dependency between the two projects.